india_compliance/gst_india/doctype/purchase_reconciliation_tool/purchase_reconciliation_tool.js

```diff
@@ -68,6 +68,9 @@ frappe.ui.form.on(DOCTYPE, {
         new india_compliance.quick_info_popover(frm, tooltip_info);
 
         await frappe.require("purchase_reconciliation_tool.bundle.js");
+         if (!frm.doc.company) {
+            frm.doc.company = frappe.defaults.get_user_default("Company");
```

Comment:
This is good one, but it should ideally default in most cases for this value instead of default company.